### PR TITLE
Feature/2023 10/greendata benchmarks

### PR DESCRIPTION
**Pull request by wongws11**
image
@winterstein Benchmark data is now hardcoded in 0.5
How should i fetch benchmarks data?

### CODE REVIEW

**Comment by wongws11** [comment]
- Benchmark should be whole country (A query to ES)
    - Getting benchmark for last quater (Last month for now)
        - Open ticket to remind me swap that to quater
    - Do a breakdown by country and get weighting
- Catch when there is no co2pm data

**Comment by wongws11** [comment]
Now it will fetch Last month's co2pm data normalised with the countries it ran on as the benchmarks. @winterstein @MichaelGoodLoop Ready to merge and test!

**Comment by wongws11** [comment]
image

File: src/js/components/pages/greendash/CompareCard.jsx [hunk 1]
```diff
@@ -181,71 +188,255 @@ const CampaignCard = ({baseFilters}) => {
...
+		if (!per1000) return null;
+
+		const countryData = await getCarbon({ ...props.baseFilters, breakdown: ['country{"count":"sum"}'] }).promise;
+		const countryMap = getCountryMapFiltered(countryData.by_country?.buckets, countryData.allCount);
+
+		const urlParams = getUrlVars();
+		/** TODO: Upgrade to last quarter later */
+		const period = getPeriodFromUrlParams({ ...urlParams, period: "last-month" });
+
+		const countryCo2pm = await Promise.all(
```
Comment (winterstein) [inline src/js/components/pages/greendash/CompareCard.jsx:358]: Hello @wongws11 
Good work. Could you add some more comments to this patch of code?
We're getting country data to do a weighted sum. Let's make that clear with comments. Thanks!

Comment (wongws11) [reply src/js/components/pages/greendash/CompareCard.jsx:358]: No problem. Will do.

**Pull request merged**